sys-auth/fprintd/fprintd-1.92.0.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit meson pam python-any-r1 systemd
+
+MY_P="${PN}-v${PV}"
+
+DESCRIPTION="D-Bus service to access fingerprint readers"
+HOMEPAGE="https://gitlab.freedesktop.org/libfprint/fprintd"
+SRC_URI="https://gitlab.freedesktop.org/libfprint/${PN}/-/archive/v${PV}/${MY_P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~sparc ~x86"
+IUSE="doc pam systemd test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/dbus-glib
+	dev-libs/glib:2
+	sys-auth/libfprint:2
+	sys-auth/polkit
+	pam? (
+		sys-libs/pam
+		systemd? ( sys-apps/systemd )
+		!systemd? ( sys-auth/elogind )
+	)"
+
+DEPEND="
+	${RDEPEND}
+	test? (
+		$(python_gen_any_dep '
+			dev-python/dbusmock[${PYTHON_USEDEP}]
+			dev-python/dbus-python[${PYTHON_USEDEP}]
+			dev-python/pycairo[${PYTHON_USEDEP}]
+		')
+		pam? ( sys-libs/pam_wrapper )
```

Comment:
[${PYTHON_USEDEP}], yes?

You also need to update python_check_deps if using pam.

Reply:
Done